.gitlab-ci.yml

```diff
@@ -890,15 +874,13 @@ test_backend_integration:
                 -f=../docker-compose.enterprise.yml
                 -f=../docker-compose.storage.minio.yml
                 -f=../extra/recaptcha-testing/tenantadm-test-recaptcha-conf.yml
-                -f=../extra/smtp-testing/conductor-workers-smtp-test.yml
                 -f=../extra/stripe-testing/stripe-test.docker-compose.yml ;;
```

Comment:
Question: is this file newer than 2.3.0? I think so. In that case, this removal can stay (here and below). But the rest of this PR needs to be reverted.

Reply:
No, it is not newer than 2.3.0